client/packages/system/src/Encounter/ListView/ListView.tsx

```diff
@@ -58,15 +57,7 @@ const EncounterListComponent: FC = () => {
 };
 
 export const EncounterListView: FC = () => (
-  <TableProvider
-    createStore={createTableStore}
-    queryParamsStore={createQueryParamsStore<EncounterFragment>({
-      initialSortBy: {
-        key: EncounterSortFieldInput.StartDatetime,
-        isDesc: true,
-      },
-    })}
-  >
```

Comment:
Removing it since the `EncounterListComponent` is no longer consuming query param store resource(s) now.

Reply:
Thanks this wasn't needed here (see other comment)